components/tools/OmeroPy/src/omero/util/upgrade_check.py

```diff
@@ -91,6 +91,17 @@ def _set(self, results, e):
         self.upgradeUrl = results
         self.exc = e
 
+    def getOSVersion(self):
+        try:
+            if len(platform.mac_ver()[0]) > 0:
+                version = "%s-%s" % (platform.platform(),
```

Comment:
I thought we were going to use `;` as the separator here so we could use a regex to separate the information?